install/linux/usr/share/odemis/sim/sparc2-ded-sim.odm.yaml

```diff
@@ -0,0 +1,338 @@
+# Configuration for the SPARC v2 system with focus simulated on ccd camera and a dedicated camera for the acquisition of the spectra
```

Comment:
This file (sparc2-ded-sim.odm.yaml) doesn't seem used anywhere. It feels that 2 simulator files should be suficient anyway, so maybe you can drop that one, unless you need it really for something specific? Actually, now that the simcam has been modified, you should be able to even use a single simulator file.